api/python/provisioner/commands/auto_deploy.py

```diff
@@ -51,6 +52,11 @@ class AutoDeploy(SetupCmdBase, CommandParserFillerMixin):
 
     @staticmethod
     def deployment_validations(deploy_check):
+
+        # even if cortx-py-util package is installed during runtime
+        # it failed to import, reload of site pkg helped here
+        importlib.reload(site)
```

Comment:
This happens due to chicken and egg problem.
We are attempting to load cortx-py-utils before installing cortx-utils rpm.
I believe we need to include any cortx-py-utils imports in try-except as well to avoid such situations.